processing/data_collection/gazette/spiders/pi_teresina.py

```diff
@@ -0,0 +1,46 @@
+import scrapy
+import dateparser
+
+from datetime import datetime
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
```

Comment:
**thought**: Unfortunately `black` didn't change the order of the imports. I have been using [isort](https://github.com/PyCQA/isort) before running `black` to order the imports following `pep8`. The result is not always compatible with `black` (the later groups some imports in a different way), but if we execute in order `isort`and then `black` the result is quite good. Maybe this could be added to some pre-commit hook in the project?

Reply:
@jvanz The requested changes have been made. Thanks for the feedback!

Reply:
@rennerocha This idea is very good and avoids future mistakes like this. I suggest opening a issue requesting this modification

Reply:
@Jefersonalves, good. idea. Can you do it? :smile:

Reply:
Issue created [241](https://github.com/okfn-brasil/querido-diario/issues/241)